modules/ROOT/pages/hybrid-manage-props.adoc

```diff
@@ -29,11 +46,11 @@ You can override the property values in the `mule-app.properties` file bundled i
 . Click *Manage Application*.
 . On the *Settings* page, click the *Properties* tab:
 +
-.The screenshot shows (*1*) the *Properties* tab, (*2*) the *Key* and *Value* fields, and (*3*) the *...* menu on the application  *Settings* page.
-image::hybrid-app-props.png[*Properties* tab, *Key* and *Value* fields, and *...* menu on the *Settings* page]
+.The screenshot shows (*1*) the *Properties* tab, (*2*) the *Key* and *Value* fields, and (*3*) the *...* menu on the application *Settings* page.
```

Comment:
Are the numbers supposed to be bold? I don't think I have seen Robert do this; dunno' if he has strong feelings about it.

Reply:
I've been putting them in bold.